pkg/tenant/tenants_test.go

```diff
@@ -14,19 +16,78 @@ func (suite *tenantTestSuite) TestTenantCreate() {
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
 
+	// Connect to a mock trtl database
 	trtl := db.GetMock()
 	defer trtl.Reset()
 
+	// Call the OnPut method and return a PutReply
 	trtl.OnPut = func(ctx context.Context, pr *pb.PutRequest) (*pb.PutReply, error) {
 		return &pb.PutReply{}, nil
 	}
 
+	// Create a tenant test fixture
 	req := &api.Tenant{
 		Name:            "tenant01",
 		EnvironmentType: "prod",
 	}
+
 	tenant, err := suite.client.TenantCreate(ctx, req)
 	require.NoError(err, "could not add tenant")
 	require.Equal(req.Name, tenant.Name, "tenant name should match")
 	require.Equal(req.EnvironmentType, tenant.EnvironmentType, "tenant id should match")
 }
+
+func (suite *tenantTestSuite) TestTenantDetail() {
+	require := suite.Require()
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	tenantID := "01ARZ3NDEKTSV4RRFFQ69G5FAV"
+	defer cancel()
+
+	// Connect to a mock trtl database
+	trtl := db.GetMock()
+	defer trtl.Reset()
+
+	// Get JSON test data.
+	data, err := os.ReadFile("testdata/tenant.json")
+	if err != nil {
+		return
+	}
```

Comment:
Can we change this to a `require.NoError`? I think as the test is written currently, if we get an error here we will just return out of the test and the rest of the test code is skipped, which makes it look like the test has passed.